src/utils/temp.ts

```diff
@@ -0,0 +1,122 @@
+import * as uuid from 'uuid';
+import neo4j, { Driver, QueryResult, Session } from 'neo4j-driver';
+
```

Comment:
You can create a `const tempDbPrefix = 'temp--';` and use it in both places in this file.

Reply:
Good catch

Reply:
Done.